Project4/styles.css

```diff
@@ -0,0 +1,151 @@
+@import url(https://fonts.googleapis.com/css?family=Montserrat);
+
+$nav-height: 60px;
+h1 h2 h3 {
+  color: #ffc400;
+}
+body {
+  text-align:center;
+  margin:0px;
+  padding:0px;
+  font-family:'Montserrat', 'Oswald', 'Poppins', 'PT Sans',
+  'Titillium Web', 'Nunito', 'PT Sans Narrow',
+  'Rubik', 'Arimo', 'Arvo', 'Cabin', 'Fjalla One',
+  'Abel', 'Kanit', san-serif;
+  color:#001d1f;
+}
+p {
+  text-align:left;
+  margin:0px;
+  padding:20px;
+  font-family:'Montserrat', 'Oswald', 'Poppins', 'PT Sans',
+  'Titillium Web', 'Nunito', 'PT Sans Narrow',
+  'Rubik', 'Arimo', 'Arvo', 'Cabin', 'Fjalla One',
+  'Abel', 'Kanit', san-serif;
+  color:#00ffaa;
+}
+a {
+  color: #061d50
+}
+.header {
+  width:100%;
+  height:100px;
+  font-size:50px;
+  line-height:100px;
+  text-transform:uppercase;
+  font-weight:bold;
+  color: #00ffff
+}
+.button1 {
```

Comment:
This is not a great classname -- generally good to avoid numbering your classnames because it's difficult to differentiate button1 from button2.